app/backend/src/clj/teet/user/user_db.clj

```diff
@@ -50,3 +51,30 @@
              [(missing? $ ?p :permission/valid-until)]]
            db user-id)
       (mapv first)))
+
+(def user-query-rules
+  '[[(user-by-name ?u ?search)
+     [?u :user/id _]
+     [?u :user/given-name ?given]
+     [(missing? $ ?u :user/deactivated?)]
+     [?u :user/family-name ?family]
+     [(str ?given " " ?family) ?full-name]
+     [(.toLowerCase ?full-name) ?full-name-lower]
+     [(.contains ^String ?full-name-lower ?search)]]])
+
+(defn find-user-by-name [db name]
+  (d/q '[:find (pull ?u [:db/id :user/id :user/given-name :user/family-name :user/email :user/person-id])
```

Comment:
Different styles for (d/q ) function in one namespace are used. For example, user-info function put :user/person-id :db/id on new line, and this function put all keys in one line. The same style should be used across namespace (at least)

Reply:
They are on a different line there just because the line gets too long. I don't think we have followed any guidelines as the what attributes of the pull should be on different lines.